src/gui/properties/speedwidget.cpp

```diff
@@ -116,53 +116,42 @@ SpeedWidget::SpeedWidget(PropertiesWidget *parent)
 
     loadSettings();
 
-    m_isUpdating = true;
-    m_updateFuture = QtConcurrent::run(this, &SpeedWidget::update);
+    m_updateTimer=new QTimer(this);
```

Comment:
please add spaces around `=`

Reply:
Also no need for `m_updateTimer` being a class memeber.
Just `QTimer *updateTimer = new QTimer(this);` is sufficient.

Reply:
fixed